aiida/cmdline/commands/cmd_code.py

```diff
@@ -31,70 +29,53 @@ def verdi_code():
     pass
 
 
-def is_on_computer(ctx):
-    return bool(ctx.params.get('on_computer'))
+def get_default(key, ctx):
+    """
+    Get the default argument using a user instance property
+    :param value: The name of the property to use
+    :param ctx: The click context (which will be used to get the user)
+    :return: The default value, or None
+    """
+    try:
+        value = getattr(ctx.code_builder, key)
+        if value == "":
+            value = None
+    except KeyError:
+        value = None
 
+    return value
 
-def is_not_on_computer(ctx):
-    return bool(not is_on_computer(ctx))
 
+def get_computer_name(ctx):
+    return getattr(ctx.code_builder, 'computer').name
 
-# Reusable options (code-specific)
-ON_COMPUTER = OverridableOption(
-    '--on-computer/--store-in-db',
-    is_eager=False,
-    default=True,
-    cls=InteractiveOption,
-    prompt='Installed on target computer?')
 
-REMOTE_ABS_PATH = OverridableOption(
-    '--remote-abs-path',
-    prompt='Remote absolute path',
-    required_fn=is_on_computer,
-    prompt_fn=is_on_computer,
-    type=types.AbsolutePathParamType(dir_okay=False),
-    cls=InteractiveOption,
-    help=('[if --on-computer]: the absolute path to the executable on the remote machine'))
+def get_on_computer(ctx):
+    return not getattr(ctx.code_builder, 'is_local')()
 
-CODE_FOLDER = OverridableOption(
-    '--code-folder',
-    prompt='Local directory containing the code',
-    required_fn=is_not_on_computer,
-    prompt_fn=is_not_on_computer,
-    type=click.Path(file_okay=False, exists=True, readable=True),
-    cls=InteractiveOption,
-    help=('[if --store-in-db]: directory containing the executable and all other files necessary for running it'))
 
-CODE_REL_PATH = OverridableOption(
-    '--code-rel-path',
-    prompt='Relative path of executable inside code folder',
-    required_fn=is_not_on_computer,
-    prompt_fn=is_not_on_computer,
-    type=click.Path(dir_okay=False),
-    cls=InteractiveOption,
-    help=('[if --store-in-db]: relative path of the executable inside the code-folder'))
+# pylint: disable=unused-argument
```

Comment:
As @dev-zero pointed out, this applies not only to the function below but to the rest of the file. I don't know if this is the right thing to do